### PR TITLE
Add docs on centralized monitoring for Kommander

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Docs ticket: https://jira.mesosphere.com/browse/DCOS-62414

Eng tickets:
https://jira.mesosphere.com/browse/DCOS-62060
https://jira.mesosphere.com/browse/DCOS-62061

## Description of changes being made
Add docs on centralized monitoring (Thanos/Grafana/Karma) in kommander

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/latest/centralized-monitoring/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+{
+    "annotations":
+    ...
+    "title": "Some Dashboard",
+    "uid": "abcd1234",
+    "version": 1
+}
+```
+
+After you decide how to create your custom dashboard, you can configure it when deploying Kommander by modifying the `cluster.yaml` file as follows:
```
[the remark this answers is not in the document]
Comment (gracedo) [reply pages/ksphere/kommander/latest/centralized-monitoring/index.md]: hehe, thanks, I stole this section from konvoy monitoring docs :p

File: pages/ksphere/kommander/latest/centralized-monitoring/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+menuWeight: 7
+excerpt: Monitor clusters created with Kommander or any connected Konvoy cluster
+---
+
+Kommander provides centralized monitoring in a multi-cluster environment using the monitoring stack running in managed Konvoy clusters.
+Centralized monitoring is provided by default in every Kommander cluster.
+
+Managed Konvoy clusters are distinguished by a cluster ID.
+This cluster ID correlates to the kube-system namespace UID of the cluster.
+You can run this kubectl command, **using the correct cluster's context or kubeconfig**, to look up the cluster's kube-system namespace UID to determine which cluster the metrics and alerts correspond to:
```
[the remark this answers is not in the document]
Comment (gracedo) [reply pages/ksphere/kommander/latest/centralized-monitoring/index.md]: We also have this ticket https://jira.mesosphere.com/browse/DCOS-62063 which should hopefully make it easier to identify clusters

**Comment by samvantran** [comment]
Hey @russurquhart1 this is the doc work for centralized monitoring in Kommander for ticket [DCOS-62414](https://jira.mesosphere.com/browse/DCOS-62414)

File: pages/ksphere/kommander/latest/centralized-monitoring/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+## Centralized Metrics
+
+Metrics are aggregated from all managed Konvoy clusters up to the Kommander cluster using Thanos.
+You can visualize these metrics in Grafana using a set of provided dashboards.
+The [Thanos Query][thanos_query] component is installed on the Kommander cluster and set up to query the Prometheus instances running in the managed clusters via a Thanos sidecar running alongside each Prometheus container.
+Grafana is configured with Thanos Query as its datasource, and comes with a pre-installed dashboard for a global view of all managed clusters, named `Kubernetes / Compute Resources / Clusters `.
+The `Thanos Query` dashboard is also installed by default to monitor the Thanos Query component.
+
+You can access the centralized Grafana UI at `https://<CLUSTER_URL>/ops/portal/kommander/monitoring/grafana`.
```
Comment (GoelDeepak) [inline pages/ksphere/kommander/latest/centralized-monitoring/index.md]: nit: is it possible to put all the URLs in their own box when you render the markup? That way it is easier to spot these URLs

**Pull request merged**